File: EasyTalentos.Dominio/Entity/Disponibilidade.cs
using System.Text.Json.Serialization;


namespace EasyTalentos.Dominio.Entity
{
    public class Disponibilidade
    {
        public int Id { get; set; }

        public bool AteQuatro { get; set; }
        public bool QuatroAteSeis { get; set; }
        public bool SeisAteOito { get; set; }
        public bool AcimaDeOito { get; set; }
        public bool FinaisDeSemana { get; set; }

        public int CandidatoId { get; set; }


        [JsonIgnore]
        public virtual Candidato Candidato { get; set; }
    }
}

File: EasyTalentos.Web/ClientApp/src/app/models/Disponibilidade.ts
import { Candidato } from "./Candidato";

export class Disponibilidade {
  id: number;
  ateQuatro: boolean = false;
  quatroAteSeis: boolean = false;
  seisAteOito: boolean = false;
  acimaDeOito: boolean = false;
  finaisDeSemana: boolean = false;
  candidatoId: number;
  candidato: Candidato;
}
